packages/@okta/vuepress-site/docs/reference/token-hook/index.md

```diff
@@ -697,17 +714,22 @@ PATCH command that removes the employee's email:
    ]
 }
 ```
+
 The resulting JSON object:
 
 ```json
-{  
-   "employee_profile":{  
+{
+   "employee_profile":{
       "employee_id":"1234",
       "name":"Anna"
    }
 }
 ```
 
+## Timeout behavior
+
+After receiving the Okta request, if there is a response timeout, the Okta process flow proceeds with original token returned. See [Troubleshooting](#Troubleshooting).
```

Comment:
I think the anchor link is #troubleshooting instead of #Troubleshooting

Reply:
Thanks! Fixed the link. The problem with the layouts for the other Hook APIs should now be gone after merging in the master branch, which had new page layouts added since I created this branch.